eth/gasprice/gasprice.go

```diff
@@ -101,21 +100,14 @@ func NewOracle(backend OracleBackend, params Config) *Oracle {
 		log.Warn("Sanitizing invalid gasprice oracle price cap", "provided", params.MaxPrice, "updated", maxPrice)
 	}
 
-	// (PIP-35): Enforce the ignore price for amoy. The default value for amoy (which is
-	// an indicator of being on amoy) should be set by the caller.
+	// PIP-35: Enforce the ignore price to 25 gwei
 	ignorePrice := params.IgnorePrice
-	if DefaultIgnorePriceAmoy != nil {
-		if ignorePrice == nil || ignorePrice.Int64() != DefaultIgnorePriceAmoy.Int64() {
-			ignorePrice = DefaultIgnorePriceAmoy
-			log.Warn("Sanitizing invalid gasprice oracle ignore price", "provided", params.IgnorePrice, "updated", ignorePrice)
-		}
-	} else {
-		if ignorePrice == nil || ignorePrice.Int64() <= 0 {
-			ignorePrice = DefaultIgnorePrice
-			log.Warn("Sanitizing invalid gasprice oracle ignore price", "provided", params.IgnorePrice, "updated", ignorePrice)
-		}
+	if ignorePrice == nil || ignorePrice.Int64() != DefaultIgnorePrice.Int64() {
+		ignorePrice = DefaultIgnorePrice
+		log.Warn("Sanitizing invalid gasprice oracle ignore price", "provided", params.IgnorePrice, "updated", ignorePrice)
+	} else if ignorePrice.Int64() > 0 {
+		log.Info("Gasprice oracle is ignoring threshold set", "threshold", ignorePrice)
```

Comment:
I see that this log was not in any condition initially. Now it is in an `else if`. Is this intentional?

Reply:
Changed to `else`, thanks!